webpack.config.js

```diff
@@ -27,6 +27,7 @@ const webpackConfig = {
         maelstromTheme: './src/plugins/themes/maelstrom-theme.scss'
     },
     output: {
+        publicPath: '',
```

Comment:
Is this needed?

Reply:
yes without it `npm install` fails.
Error: Automatic publicPath is not supported in this browser

Reply:
Actually, can you try to upgrade the mini-css-extract-plugin to 1.3.9 and remove the publicPath? I think that fixes the issue.